testrunner/src/mock-function/lib/insert.test.ts

```diff
@@ -0,0 +1,143 @@
+// insertDocument.test.ts
+import { describe, it, beforeEach, mock } from 'node:test';
+import assert from 'node:assert';
+import { Container } from '../data/connect-to-cosmos';
+import { createTestInputAndResult } from '../data/fake-data';
+import {
+  DbDocument,
+  DbError,
+  isDbError,
+  isVerificationErrors,
+  RawInput,
+} from '../data/model';
+// Instead of jest.mock, import the whole module to override functions as needed.
+import * as verifyModule from '../data/verify';
+import * as dbModule from '../data/connect-to-cosmos';
+import { insertDocument } from './insert';
```

Comment:
```suggestion
import { Container } from '../data/connect-to-cosmos.ts';
import { createTestInputAndResult } from '../data/fake-data.ts';
import type {
  DbDocument,
  DbError,
  isDbError,
  isVerificationErrors,
  RawInput,
} from '../data/model.ts';
// Instead of jest.mock, import the whole module to override functions as needed.
import * as verifyModule from '../data/verify.ts';
import * as dbModule from '../data/connect-to-cosmos.ts';
import { insertDocument } from './insert.ts';
```